src/components/HomepageFeatures/index.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import clsx from 'clsx';
+import styles from './styles.module.css';
+
+const FeatureList = [
```

Comment:
Can these please be changed to the UOR pillars? https://docs.google.com/presentation/d/e/2PACX-1vSBbrsaCHP4irxT4s41YJREdEeAtvdgX8xrMwNpOh_hBZa6eKybUpD3FLr06f9QCKKSp9-Hgs47RIUC/pub?start=false&loop=false&delayms=10000&slide=id.g1445db85d7c_0_238

Reply:
resolved in [7f31d4f](https://github.com/uor-framework/universalreference.io/pull/11/commits/7f31d4ffddba6c100941e3d67a841925fee8f5d1)